### ISSUE TITLE
Some errors on 2.1.0

### ISSUE THREAD

**Issue by Benjaminsson** [podlove/podlove-publisher#733]
Hi,

I got some error on the latest update. This error occurs when looking on the dashboard for the network admin:
(...)/public_html/wp-content/plugins/podlove-podcasting-plugin-for-wordpress/lib/modules/contributors/contributors.php on line 374 

And i get this error when trying to create a new template:
Fatal error: Call to undefined method Podlove\Modules\Networks\Settings\Templates::new_template() in (...)/public_html/wp-content/plugins/podlove-podcasting-plugin-for-wordpress/lib/modules/networks/settings/templates.php on line 81

Thanks for an otherwise very cool update!

**Comment by eteubert** [comment]
> This error occurs when looking on the dashboard for the network admin:
> (...)/public_html/wp-content/plugins/podlove-podcasting-plugin-for-wordpress/lib/modules/contributors/contributors.php on line 374

What's the error? You only posted the line where it happens.

> error when trying to create a new template

That button should not even exist, see screenshot. Will be removed soon.

![templates_ _network_admin__podlove-publisher_-_network_ _wordpress](https://cloud.githubusercontent.com/assets/235918/7360364/83ea1f54-ed7c-11e4-94a6-eb9f5ee3baf0.png)

**Comment by Benjaminsson** [comment]
> What's the error?
Oops, it says 
```
Warning: Division by zero in (...)/public_html/wp-content/plugins/podlove-podcasting-plugin-for-wordpress/lib/modules/contributors/contributors.php on line 374 Warning: Division by zero in (...)/public_html/wp-content/plugins/podlove-podcasting-plugin-for-wordpress/lib/modules/contributors/contributors.php on line 374 Warning: Division by zero in (...)/public_html/wp-content/plugins/podlove-podcasting-plugin-for-wordpress/lib/modules/contributors/contributors.php on line 374
```

> That button should not even exist, see screenshot. Will be removed soon.
I can't add it with that button either. It gets added and everything but when i save i get this message:
`Error: Could not save template.` on line 111 in template.js

**Comment by eteubert** [comment]
Will be fixed in 2.1.3
